### ISSUE TITLE
[Musicxml Export] - Direction in different voices are all in voice 1

### ISSUE THREAD

**Issue by hhpmusic** [musescore/MuseScore#25663]
### Issue type

Import/export issue

### Description with steps to reproduce

This was raised before, but probably on the old site. In this example, texts v1 and v2 should be exported into corresponding voices instead of all in voice 1. All other directions like dynamic, wedge and brackets/dashes should all in this case. Otherwise in braille they are also translated in voice 1, and I'll not able to determine where they should be, unless looking into the xml source.

### Supporting files, videos and screenshots

[voice text.zip](https://github.com/user-attachments/files/17932737/voice.text.zip)

### What is the latest version of MuseScore Studio where this issue is present?

4.4

### Regression

No.

### Operating system

Windows 11

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by rettinghaus** [comment]
I'm not sure about this. `staffText` in MuseScore always apply to all voices (that's why it is "staff text"). Expressions and dynamics now can be assigned to voices, but I'm not sure how to express this correctly in MusicXML.

**Comment by hhpmusic** [comment]
No, I find these texts are in different voices depending on when they are input, like my example. Stafftext applies to the voice it's entered, like in Sibelius. If not considering this, please just translate the direction tag where the text or other things locate, instead of merging them all into voice 1.

**Comment by hhpmusic** [comment]
> I'm not sure about this. `staffText` in MuseScore always apply to all voices (that's why it is "staff text"). Expressions and dynamics now can be assigned to voices, but I'm not sure how to express this correctly in MusicXML.

Here's the correct description of all direction objects, including text, line and other directional objects, attached to the same voice in Musescore.

[voice text xml.zip](https://github.com/user-attachments/files/27472978/voice.text.xml.zip)
